src/Our.Umbraco.Ditto/ComponentModel/Processors/DittoProcessorRegistry.cs

```diff
@@ -141,6 +148,36 @@ public IEnumerable<DittoProcessorAttribute> GetRegisteredProcessorAttributesFor(
                     ? Cache[objectType]
                     : Enumerable.Empty<DittoProcessorAttribute>();
             }
-        }        
+        }
+
+        /// <summary>
+        /// Registers a processor attribute to the end of the default set of post-processor attributes.
+        /// </summary>
+        /// <typeparam name="TProcessorAttributeType"></typeparam>
+        /// <param name="position"></param>
+        public void RegisterPostProcessorAttribute<TProcessorAttributeType>(int position = -1)
+            where TProcessorAttributeType : DittoProcessorAttribute, new()
+        {
+            var processor = (DittoProcessorAttribute)typeof(TProcessorAttributeType).GetInstance();
+
+            if (position < 0)
+            {
+                this.DefaultPostProcessorAttributes.Add(processor);
+            }
+            else
+            {
+                this.DefaultPostProcessorAttributes.Insert(position, processor);
+            }
+        }
+
+        /// <summary>
+        /// Unregisters a processor attribute from the default set of post-processor attributes.
+        /// </summary>
+        /// <typeparam name="TProcessorAttributeType"></typeparam>
+        public void UnregisterPostProcessorAttribute<TProcessorAttributeType>()
```

Comment:
DeregisterProcessorAttribute?

Reply:
Agreed, with the thinking being: "deregister" is the act and "unregister" is the state